CHANGELOG.md

```diff
@@ -5,6 +5,41 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+- `app-cirrus-${CIRRUS_DEPLOYMENT}-workflow-event` SNS topic, and
+  `WorkflowEventManager` class for managing workflow event actions. ([#261])
+  The actions managed by this class include:
+
+  - updating state of workflows in `StateDB`
+  - announcing interactions cirrus has with a payload to the
+    `workflow-event-topic`
```

Comment:
one more `-topic`

Reply:
Got that, and fixed the fq-name format to the default `cirrus-<stage>-workflow-event`